### PR TITLE
Keymap for Idobo

### PR DESCRIPTION
**Pull request by GreenShadowMaker**
## Description

New Keymap for Idobo

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

N/A

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/idobo/keymaps/greenshadowmaker/keymap.c [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+-----------------+--------|
+ * | RAISE  | A      | S      | D      | F      | G      |        | UP     |        | H      | J      | K      | L      | ;      | '      |
+ * |--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------+--------------------------+--------|
+ * | LSHIFT | Z      | X      | C      | V      | B      | LEFT   | DOWN   | RIGHT  | N      | M      | ,      | .      | /      | RSHIFT |
+ * |--------+--------+--------+--------+--------+-----------------+--------+--------+--------+--------+-----------------+--------+--------|
+ * | `      | \      | LALT   | LCTRL  | LOWER  | SPACE  | LGUI   | DEL    | ENTER  | SPACE  | RAISE  |        |        |        | PrtScr |
+ * '--------------------------------------------------------------------------------------------------------------------------------------'
+ */
+
+ [_QWERTY] = { /* QWERTY */
```
Comment (drashna) [inline keyboards/idobo/keymaps/greenshadowmaker/keymap.c]: Could you reformat this to match this: 
https://github.com/qmk/qmk_firmware/blob/master/keyboards/idobo/keymaps/default/keymap.c#L25-L31

Comment (GreenShadowMaker) [reply keyboards/idobo/keymaps/greenshadowmaker/keymap.c]: updated, thanks!!  I missed that the style was different from xd75/default and idobo/default (confusion around board name from massdrop) and had copy/pasted

Comment (drashna) [reply keyboards/idobo/keymaps/greenshadowmaker/keymap.c]: No worries. It's been a fun time getting people sorted on that issue!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**